README.md

```diff
@@ -60,12 +66,18 @@ The previous Team of Volenteers has left a todo list to finish cleaning the cbc
 The data are primarily flat files saved to Github since they are not the largest files. Down the road, we can consider saving them to S3 or GCS.
 
 ### Onboarding
-#### Weather Data API
-  For the weather data, DataKind needs to understand what weather data is avilible for the past based on lat long data (that we have from the cbc data file)
 
-The NOAA API should be good for this. The smallest and most precise unit in the NOAA data is for a Station. Via the NOAA api we can list of all the stations and their lat long data. I think the next task will be to write a script that finds the closest NOAA station to each of the circles in the data set.
+## Cleanest Volenteer Submitted Data
+The file bird_count_cleaned_may_29_2019.csv is the cleanest data to date and was produced by the audubon_world.ipynb file. 
+
+## Offical Weather Date: Google BigQuery and Weather data
+To optain the daily measures of weather data, we will be using the GHCN Daily database powered by Google.
+
+https://console.cloud.google.com/marketplace/details/noaa-public/ghcn-d?filter=solution-type:dataset&id=9d500d1d-fda4-4413-a789-d8786fd6592e&pli=1
```

Comment:
Nitpick: you can link inside the README by doing something like this

"To optain the daily measures of weather data, we will be using the GHCN Daily
database powered by Google, which can be found [here](https://console.cloud.google.com/marketplace/details/noaa-public/ghcn-d?filter=solution-type:dataset&id=9d500d1d-fda4-4413-a789-d8786fd6592e&pli=1)"